ui/components/multichain/network-list-menu/network-list-menu.test.js

```diff
@@ -150,4 +158,22 @@ describe('NetworkListMenu', () => {
       document.querySelectorAll('multichain-network-list-item__delete'),
     ).toHaveLength(0);
   });
+
+  it('fires setNetworkClientIdForDomain when network item is clicked', () => {
+    const { getByText } = render();
+    fireEvent.click(getByText(MAINNET_DISPLAY_NAME));
+    expect(mockSetNetworkClientIdForDomain).toHaveBeenCalledWith(
+      MOCK_ORIGIN,
+      NETWORK_TYPES.MAINNET,
+    );
+  });
+
+  it('fires setNetworkClientIdForDomain when test network item is clicked', () => {
+    const { getByText } = render({ showTestNetworks: true });
+    fireEvent.click(getByText(SEPOLIA_DISPLAY_NAME));
+    expect(mockSetNetworkClientIdForDomain).toHaveBeenCalledWith(
+      MOCK_ORIGIN,
+      NETWORK_TYPES.SEPOLIA,
```

Comment:
i guess we should add this code on testnet ( see it [here](https://github.com/MetaMask/metamask-extension/pull/25259/commits/bc8a7fa6a236423ebccc48f0010da067c13c4c34#diff-1832ff00c884a64a41fb0eabeb10d3472192e1f5dd4a3e36018f5d4706944bd0R312-R317) )

Reply:
for more context see the bug below:

https://github.com/MetaMask/metamask-extension/assets/26223211/a940295c-c71f-4aff-8c40-65769978055c